app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -25,11 +25,21 @@ def self.rrd_claim_processed?(submission)
       submission.form_json.include? RapidReadyForDecision::HypertensionUploadManager::DOCUMENT_TITLE
     end
 
+    # Fetch all claims from EVSS and return whether there are any open EP 020's.
+    # This method could be moved into a Concern when ProcessorSelector adds new job classes.
+    def self.pending_eps?(auth_headers)
+      all_claims = EVSS::ClaimsService.new(auth_headers).all_claims.body
+      all_claims['open_claims'].any? { |claim| claim['base_end_product_code'] == '020' }
+    end
+
     def perform(form526_submission_id)
       form526_submission = Form526Submission.find(form526_submission_id)
 
       begin
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
+          # Prefer "next" keyword here so that after-yield code continues execution
```

Comment:
It's more than a preference:
```suggestion
          # Use "next" keyword here so that after-yield code continues execution and job will not have a `try` status
```

Reply:
Will fix in a separate PR